core-bundle/src/EventListener/SubrequestCacheListener.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace Contao\CoreBundle\EventListener;
+
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\HttpCache\ResponseCacheStrategy;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class SubrequestCacheListener
+{
+    public const MERGE_CACHE_HEADER = 'Sf-Merge-Cache-Control';
```

Comment:
Imho this should be `Contao-Merge-Cache-Control`.

Reply:
changed in e50d1d864674632ced16783d6731d4f37832cf00